example_parlai_internal/README.md

```diff
@@ -0,0 +1,65 @@
+# Internal subfolder
+
+By creating an internal subfolder, you can set up your own custom agents and tasks,
+create your own model zoo, and even manage it all with a separate git repository.
+We've set it up so you can now work inside the ParlAI folder without the risk of accidentally pushing your work to the public ParlAI repo.
+We've also added some convenient shortcuts to mirror what we provide in the `parlai` folder.
+
+## How to
+
+Start by creating a new folder named parlai_internal and copying the contents of this folder.
+ (We've added this to .gitignore already.)
+
+```bash
+cd ~/ParlAI
+mkdir parlai_internal
+cp -r example_parlai_internal/* parlai_internal
```

Comment:
is this right? I would think the `/*` shouldn't be there.

Reply:
both work, I'll get rid of the * since I guess that's more typical?

Reply:
oh I misread, the backslash is needed otherwise you'll end up with parlai_internal/example_parlai_internal